index.yaml

```diff
@@ -552,13 +553,14 @@ entries:
       catalog.cattle.io/namespace: longhorn-system
       catalog.cattle.io/os: linux
       catalog.cattle.io/provides-gvr: longhorn.io/v1beta1
+      catalog.cattle.io/rancher-version: < 2.6.1-0
```

Comment:
To Reviewers:
The annotation is added to all applied charts. 
The rest of the changes are timestamps and digest value which we can ignore.